content/webapp/services/prismic/fetch/articles.ts

```diff
@@ -2,7 +2,19 @@ import { Query } from '@prismicio/types';
 import { GetServerSidePropsPrismicClient } from '.';
 import { ArticlePrismicDocument } from '../articles';
 
+// This is for the outro links
+const documentLinkTypes = [
+  'pages',
+  'event-series',
+  'books',
+  'events',
+  'articles',
+  'exhibitions',
+  'series',
+].flatMap(type => [`${type}.title`, `${type}.promo`]);
+
 const fetchLinks = [
+  ...documentLinkTypes,
   'series.title',
```

Comment:
```suggestion
  ...documentLinkTypes,
```

I think `series.title` is being duplicated here?